### ISSUE TITLE
EPI-SEM-001: Separate statistical strength from crypto/reproducibility authority

### ISSUE THREAD

**Issue by Tristan-Stoltz-ERC** [Luminous-Dynamics/symthaea#5283]
## Problem

While grounding MEL-EPI-001 (#5275) in the existing `symthaea-epistemic-types` crate, an important semantic collision surfaced.

The current E-axis names include:

```text
E3CryptographicallyProven
E4PubliclyReproducible
```

but several current producers assign those tiers from evidence count, z-score, or runtime-cycle count rather than from cryptographic verification or public reproducibility evidence.

Examples currently visible in source include:

- statistical retrieval mapping z-score bands to `E3CryptographicallyProven` / `E4PubliclyReproducible`;
- causal-explanation logic describing a counterfactual/statistical proof as `E3CryptographicallyProven`;
- cognitive-loop logic promoting cycle counts to `E3CryptographicallyProven`.

Therefore the present label can overstate what the underlying producer actually establishes.

## Required separation

```text
statistical strength
!= cryptographic authentication/proof
!= independent reproducibility
!= public reproducibility
```

MEL-EPI claim scopes must **not** inherit authority from the ordinal E-axis labels.

## Proposed repair program

1. Audit every producer of `EmpiricalTier` / `EmpiricalLevel`.
2. Split descriptive statistical-strength/confidence classification from provenance/verification properties.
3. Make cryptographic verification an explicit evidence/receipt property, never an inference from z-score, sample count, or cycle count.
4. Make reproducibility an explicit demonstrated property tied to exact artifacts/environment/execution evidence, never an inference from significance.
5. Add migration adapters if the E0-E4 vocabulary must remain serialized for compatibility.
6. Add regressions proving statistical evidence alone cannot mint crypto/reproducibility claims.

## Relationship to MEL-EPI-001

The new Melothaea epistemic kernel should use positive namespaced claim scopes and exact provenance instead of interpreting `EmpiricalLevel` ordinal values as scientific authority.

This issue should be repaired independently; #5275 must not silently redefine legacy E/N/M semantics or use them as a shortcut.

## Non-goal

This issue does not choose whether the legacy E0-E4 scale remains useful as a heuristic/UI classification. It only prevents its labels from being treated as evidence of properties that were not actually demonstrated.

**Comment by Tristan-Stoltz-ERC** [comment]
Deeper implementation pass has now split EPI-SEM-001 into explicit bounded lineages rather than a repo-wide enum rename.

Current work:

- #5331 `EPI-SEM-001A-R2` — cross-system semantic inventory. Fresh direct child of `main`; audits E3/E4 aliases, proxy-to-authority producers, propagation/gating seams, N/M definition collisions, and the already-hardened physics bridge as a positive control.
- #5333 — NEVER MERGE exact-head qualifier for #5331. Exact qualification run `35574868160` has been admitted and is currently queued; queued != PASS.
- #5334 `EPI-SEM-001B` — producer repair train: HDC z-score, causal support heuristic, cycle-count maturity, Phi/Mycelix boundary, duplicated physics classifiers, then cross-producer negative authority regressions.
- #5337 `EPI-SEM-001C` — version legacy E/N/M semantics and make compatibility projection non-authoritative; explicitly separates corroboration from access/disclosure policy and epistemic durability from storage retention.

Important new findings beyond the original #5283 examples:

```text
Phi >= threshold
!= cryptographic verification

open source + simulation confidence
!= public reproduction

catalog similarity
!= consensus / axiomatic truth

numeric E/N/M compatibility
!= semantic equivalence
```

R1 #5326/#5327 was closed unqualified after these additional alias/semantic surfaces were discovered. No PASS was discarded.

Implementation ordering remains fail-closed: do not stack B1 source repair on the audit until #5333 actually executes successfully against frozen #5331.

**Comment by Tristan-Stoltz-ERC** [comment]
Authority-flow continuation update after superseding incomplete R5R3.

Current frozen discovery subject:

- product #5386 — `30fe9d0cb39fe726d6317879354b9903c1c03da1`
- tree `96ee6341d111bb59553b69e9cb29681b7c754f8f`
- exact qualifier #5387 — `80b6bcffe0c412cac2c0ca189d1568220fd6df2c`
- admitted ready-for-review run `35604647813` is currently queued; no PASS claimed.

R5R4 now separates four consumer/control surfaces:

```text
1D EpistemicStatus/ordinal gate
4D EpistemicCubeGate
strict-code E-tier gating
explicit gate controls/bypasses
```

The additional cube-gate finding is material: E3/E4 directly change assertion/hedging logits, N3 changes axiomatic framing, M3 changes foundational framing. This means producer-side repair alone is insufficient; #5385 owns the consumer firewall.

## R5R4 -> R6 discipline

Do not make R5R4 itself a frozen inventory. If and only if its exact qualifier produces `PASS_DISCOVERY`, construct R6 as a fresh direct child of unchanged `main` from the emitted artifact.

R6 must freeze, for every named surface:

- exact sorted path list;
- exact path-set SHA-256 digest;
- exact count as a redundant diagnostic, not the identity;
- required witness contract;
- discovery schema/profile identity.

R6 qualification must independently rescan the exact source tree and require equality of the full ordered sets/digests, not merely `actual_count >= N` and not a manually retyped allowlist.

```text
dynamic discovery PASS
!= frozen inventory PASS

same count
!= same files

frozen path identity + digest + witnesses
-> PASS_INVENTORY only
```

Only exact executable R6 `PASS_INVENTORY` should unlock #5334 semantic producer repairs and #5385 consumer-firewall implementation.

**Comment by Tristan-Stoltz-ERC** [comment]
R6 hardening addendum: path-set identity is necessary but should not be the only durable source identity.

R5R4's set digest intentionally hashes the ordered **path list**. Because its exact qualifier also binds an exact parent/product SHA, that is sufficient for the discovery subject itself. When constructing R6, however, mechanically derive and freeze per-path source identity too:

```text
FrozenSurfaceEntryV1 {
  path,
  git_blob_id,          # preferred repository-native identity
  sha256_bytes,         # optional cross-tool/content identity
}
```

Then freeze a surface digest over the ordered `(path, source_identity)` entries in addition to the path-set digest.

This gives R6 two distinct invariants:

```text
membership digest  -> did the same files participate?
content digest     -> are those exact source bytes still the reviewed bytes?
```

Do not make file mtime/size an authority identity. Counts remain diagnostics only. Required semantic witnesses remain a third independent guard; content identity does not prove the interpretation is correct.

**Comment by Tristan-Stoltz-ERC** [comment]
R6 inventory contract tightened while #5391 remains queued.

New repository-native precedent: `crates/domains/symthaea-fabrication-kernel/src/artifact_set.rs` already models exact inventories as canonical ordered paths with byte length + SHA-256 and rejects substitutions/extras. R6 should borrow that structure, not the fabrication-domain semantics.

## R6 measured-source identity

R5R5-R2's scanner walks only production `.rs` files and excludes `scripts/`, docs, standalone tests/examples/benches. Its measurement files therefore do not currently contaminate the Rust surface sets.

Even so, R6 must explicitly bind the source subject that was measured rather than rescan an incidental qualifier checkout:

```text
source_subject_commit = adb69f11fa8068b019cc5bb598d0c7726a197fc9
source_subject_tree   = 35a6c5fdba319556af9bb487838734f67c8ac0d6
measurement_product   = #5390 / 9d7f3cebc90ed69553d0487bd9f312c2259f5fdf
measurement_qualifier = #5391 / 9c4501835500b4c90c4f975b41c63a885d5f972b
```

R6 itself should be a fresh direct child of the exact source subject and should scan/read `source_subject_commit`, not its own added manifest/checker files.

## Proposed R6 canonical inventory shape

For each unique inventoried source file freeze at least:

```text
InventoryFileV1 {
  path,
  git_mode,
  git_blob_oid,
  byte_length,
  sha256,
}
```

For every named authority surface freeze:

```text
AuthoritySurfaceInventoryV1 {
  surface_name,
  ordered_paths,
  path_set_sha256,
  ordered_file_refs,
  content_set_sha256,
  required_witness_ids,
}
```

And at top level:

```text
EpistemicAuthorityInventoryV1 {
  schema,
  source_subject_commit,
  source_subject_tree,
  discovery_schema,
  discovery_authority_scope,
  measurement_product_commit,
  exact_discovery_output_sha256,
  exact_consumer_witness_output_sha256,
  files,
  surfaces,
  witness_contract,
}
```

The manifest digest should use a versioned domain separator over canonical bytes, following the fabrication kernel's `...digest.v1\0` pattern.

## Required distinctions

```text
same count != same paths
same paths != same bytes
same bytes != same semantic witness relation
same inventory != semantic correctness
```

Git blob OIDs are useful repository identities, but SHA-256 remains the portable content commitment. File mode is included so executable/non-executable drift cannot hide behind identical bytes.

The exact R5R5-R2 outputs are still unavailable because admitted run `35616089981` remains queued. Therefore no R6 manifest values are being guessed or manually reconstructed from search results.

## R6 qualifier requirements

A future exact qualifier should independently:

1. verify R6 parent/source-subject identity;
2. read the exact frozen source subject tree;
3. recompute every file byte length, SHA-256, Git blob identity and mode;
4. independently rerun the discovery and exact production-consumer witness programs against that same source subject;
5. require exact equality of ordered path sets, path digests, file identities and witness outcomes;
6. reject missing/extra/substituted files or surfaces;
7. recompute the domain-separated whole-manifest digest;
8. emit only `PASS_INVENTORY`.

`PASS_INVENTORY` remains a census/freeze theorem only and does not validate the legacy authority semantics.

**Comment by Tristan-Stoltz-ERC** [comment]
Additional R6 lineage rule: `PASS_INVENTORY` is exact-subject evidence and must not float across later `main` drift.

## Exact-subject theorem

```text
PASS_INVENTORY(source_subject = S)
!= PASS_INVENTORY(any descendant of S)
```

Even an apparently unrelated production Rust change may add/remove an authority-shaped pattern because the R5R5-R2 discovery scans the whole production `.rs` tree.

Therefore the clean execution model is:

```text
source subject S
  -> qualified R5R5-R2 discovery of S
  -> qualified R6 inventory of S
  -> B1 semantic repair as exact child of S (or of a deliberately frozen repair base derived from S)
```

If `main` has advanced before B1 is ready to integrate:

```text
repair patch demonstrated on S
+ newer main S'
-> transplant/reapply patch to S'
-> rerun discovery/inventory revalidation on S'
-> rerun semantic repair qualification
```

Do not interpret an old inventory as a capability authorizing changes on a new tree.

### Cheap drift diagnostics are not authority

A compare showing that inventoried files did not change is useful triage, but it is not sufficient by itself because a new/changed Rust file outside the prior inventory can introduce a new authority surface.

At minimum, transfer to S' requires rerunning the broad discovery against S' and checking the exact production-consumer witness audit again. If those outputs are byte-identical to the qualified inventory/witness contract, a small revalidation tranche can establish equivalence; otherwise create a new inventory lineage.

```text
unchanged old inventoried files
!= no new authority path
```

This keeps the audit aligned with the existing no-lineage-mixing discipline elsewhere in Symthaea.

**Comment by Tristan-Stoltz-ERC** [comment]
EPI-AUTH-FLOW execution boundary update (2026-09-22): exact qualifier #5391 has now completed successfully for R5R5-R2.

```text
product 9d7f3cebc90ed69553d0487bd9f312c2259f5fdf
qual    9c4501835500b4c90c4f975b41c63a885d5f972b
run     35616089981
job     106387046876

PASS_DISCOVERY
PASS_CONSUMER_WITNESS
```

Qualified artifact digest:

```text
e2389e8dc1700898e5d956055cc7862a14f7f723398027491c2c9cce6e64415a
```

Discovery/consumer stdout SHA-256:

```text
07cc24071bd84cb277dd91a668f1da96b29e5c4c37322b83b10b3ff195870504
26ed61bb6f4cd49ac1673b3a96e95285772417586728eb657f244e64f36fdea2
```

The qualified discovery freezes 17 surfaces / 193 memberships / 108 unique production-file paths plus seven exact production-consumer witness groups. It is still measurement-only; it does not establish positive authority semantics.

Mechanical byte-freeze R6 is now #5488 with direct-main product #5502 (`19b483b6...`, tree `8009b04e...`) and draft never-merge qualifier #5508 (`76a4232d...`). R6 replays the exact frozen R5 measurement programs and requires byte-identical stdout hashes before byte-binding source files. `PASS_INVENTORY` is **not yet established**.
